src/EPR.CommonDataService.Api.UnitTests/Controllers/SubmissionsControllerTests.cs

```diff
@@ -106,8 +106,6 @@ public async Task GetApprovedSubmissionsWithAggregatedPomData_WhenValidDateStrin
     public async Task GetApprovedSubmissionsWithAggregatedPomData_WhenNoApprovedSubmissionsForValidDate_ReturnsNotFound()
```

Comment:
Can you rename this suffix to be _ReturnsNoContent()?

Reply:
updated